sight_service/service_root.py

```diff
@@ -182,6 +193,70 @@ def get_instance(self, client_id: str) -> OptimizerInstance:
     # logging.debug("<<<<<< Out %s method of %s file.", sys._getframe().f_code.co_name, os.path.basename(__file__))
 
 
+import json
+from typing import Any, Callable, Dict, List, Optional
+
+from sight.proto import sight_pb2
+from sight_service.proto import service_pb2
+
+
+def get_proto_value_from_value(v) -> sight_pb2.Value:
```

Comment:
Can we import this from sight/utils instead of keeping an extra copy here?

Reply:
removed , already present at utils